docs/development_guide/README.md

```diff
@@ -166,7 +166,7 @@ and each change would be accompanied by a unit test.
 To run all unit tests you can use the following `Makefile` target:
 
 ```bash
-make go-tests
+make go-test
```

Comment:
I would steer people away from `make go-test` since CI actually runs `make go-test-coverage` which is a bit more selective about which tests are run. That will skip e2e tests for example, which require more setup than the plain `go test ./...` that `make go-test` invokes.
```suggestion
make go-test-coverage
```